CLI/Testing/Cxx/CMakeLists.txt

```diff
@@ -28,7 +28,7 @@ if(QINPROSTATE001)
     --compare ${QINPROSTATE001}/Baseline/phantom-ktrans.nrrd
     ${TEMP}/${testname}-ktrans.nrrd
     ModuleEntryPoint        
-                --T1Tissue 1597
+                --T1Tissue 1000
```

Comment:
Oh, good - so we already have some tests!

Why did you change the parameter here as well? This is a prostate test dataset, so it definitely should correspond to prostate tissue T1!